### DISCUSSION TITLE
🚀 Feature request How hard would it be to add ability to specify a number for image resize in rte

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3346] [Feature Requests]
### Suggest an idea for ExpressionEngine

On thing folks like about wygwam vs rte is that when you add an image, you can specify the size via h and/or w field.  In the RTE, you can drag to resize an image, but I don't see an easy way to just say 200w, which is nice to be able to do.  And heck, maybe even set some default image handling.

BOTH wygwam and rte do the 'resizing' via css/markup, so it's mostly the interface that differs.  I think rte is a nicer look, but I DO like being about to just specify the h/w as normally that's what I'd want to do anyway.

**Comment by intoeetive** [comment]
CKeditor 5 allows you to define your own pre-set image sizes (in px or %) and apply those to images. There are few slightly different UX approaches (buttons or dropdowns), but any of those would require using Advanced JSON configuration 

The documentation can be found here - JavaScript options would need to be converted to JSON format. https://ckeditor.com/docs/ckeditor5/latest/features/images/images-resizing.html

Here's what I had to add to enable `100px`, `200px` and 'Original' buttons:

```
...
"image": {
        "toolbar": [
           ...
            "resizeImage:100",
            "resizeImage:200",
            "resizeImage:original"
        ],
        "resizeUnit": "px",
        "resizeOptions": [
            {
                "name": "resizeImage:original",
                "label": "Original",
                "value": null,
                "icon": "original"
            },
            {
                "name": "resizeImage:100",
                "label": "100px",
                "value": "100",
                "icon": "medium"
            },
            {
                "name": "resizeImage:200",
                "label": "200px",
                "value": "200",
                "icon": "large"
            }
        ]
...
```

I realize a visual way to do that would be a lot easier to use, so keeping this FR

**Reply by robinsowell** [reply]
That is still mad helpful, thanks Yuri!
